nuxeo-features/nuxeo-platform-rendition/nuxeo-platform-rendition-core/src/main/java/org/nuxeo/ecm/platform/rendition/service/RenditionServiceImpl.java

```diff
@@ -517,8 +517,10 @@ private void removeStoredRenditions(Map<String, List<String>> liveDocumentRefsTo
 
                 if (!renditionRefsToDelete.isEmpty()) {
                     session.removeDocuments(
-                            renditionRefsToDelete.stream().map(IdRef::new).collect(Collectors.toList()).toArray(
-                                    new DocumentRef[renditionRefsToDelete.size()]));
+                            renditionRefsToDelete.stream()
+                                                 .map(IdRef::new)
+                                                 .collect(Collectors.toList())
+                                                 .toArray(new DocumentRef[renditionRefsToDelete.size()]));
```

Comment:
can be replaced by 
`renditionRefsToDelete.stream()
                            .map(IdRef::new)
                            .toArray(DocumentRef[]::new)
`
wdyt

Reply:
This is existing code, only formatted, but sure why not.

Reply:
thanks